packages/react/src/components/UIShell/Switcher.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState, useRef } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Couldn't find `useState`/`useRef` references.